app/src/main/java/com/example/cm/ui/adapters/OnboardingAdapter.java

```diff
@@ -0,0 +1,81 @@
+package com.example.cm.ui.adapters;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.viewpager.widget.PagerAdapter;
+
+import com.example.cm.R;
+
+public class OnboardingAdapter extends PagerAdapter {
+
+    Context context;
+    LayoutInflater layoutInflater;
+
+    public OnboardingAdapter(Context context){
+        this.context = context;
+    }
+
+    int images[] = {
+            R.drawable.ic_conversation,
+            R.drawable.ic_winke_profil,
+            R.drawable.ic_having_fun,
+            R.drawable.ic_my_location
+    };
+
+    int headings[] = {
+            R.string.start_title,
+            R.string.profile_title,
+            R.string.meet_title,
+            R.string.location_title
+    };
+
+    int descs[] = {
+            R.string.start_header,
+            R.string.profile_header,
+            R.string.meet_header,
+            R.string.location_header
+    };
+
+    @Override
+    public int getCount() {
+        return headings.length;
+    }
+
+    @Override
+    public boolean isViewFromObject(@NonNull View view, @NonNull Object object) {
+        return view == (LinearLayout) object;
+    }
+
+    @NonNull
+    @Override
+    public Object instantiateItem(@NonNull ViewGroup container, int position) {
+       layoutInflater = (LayoutInflater) context.getSystemService(context.LAYOUT_INFLATER_SERVICE);
```

Comment:
context.LAYOUT_INFLATER_SERVICE -> Context.LAYOUT_INFLATER_SERVICE

variable is static